src/content/en/updates/2017/12/audio-worklet.md

```diff
@@ -2,45 +2,40 @@ project_path: /web/_project.yaml
 {# wf_featured_snippet: Chrome 64 comes with a highly anticipated new feature in Web Audio API - AudioWorklet. AudioWorklet nicely keeps the user-supplied JavaScript code all within the audio processing thread — that is, it doesn’t have to jump over to the main thread to process audio. #}
 {# wf_blink_components: Blink>WebAudio #}
 
 # Enter AudioWorklet {: .page-title }
 
 {% include "web/_shared/contributors/hongchanchoi.html" %}
 
+Note: AudioWorklet is enabled by default in Chrome 66.
+
 Chrome 64 comes with a highly anticipated new feature in Web Audio API -
 [AudioWorklet](https://webaudio.github.io/web-audio-api/#AudioWorklet). This
 article introduces its concept and usage for those who are eager to create a
-custom audio processor with JavaScript code. Please take a look at the
-[live demos](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/)
-on GitHub or [the instruction](#experimental) on how to use this
-experimental feature in Chrome 64.
-
-<aside class="note">
-  <p>
-    <b>A little bit of background: <code>ScriptProcessorNode</code></b>
-    (and why you shouldn't use it)
-  </p>
-  <p>
-    Audio processing in Web Audio API runs in a separate thread from the main
-    UI thread, so it runs smoothly. To enable custom audio processing in
-    JavaScript, the Web Audio API proposed a ScriptProcessorNode which used
-    event handlers to invoke user script in the main UI thread.
-  </p>
-  <p>
-    There are two problems in this design: the event handling is asynchronous
-    by design, and the code execution happens on the main thread. The former
-    induces the latency, and the latter pressures the main thread that is
-    commonly crowded with various UI and DOM-related tasks causing either UI
-    to "jank" or audio to "glitch". Because of this fundamental design flaw,
-    <code>ScriptProcessorNode</code> is deprecated from the specification and
-    replaced with AudioWorklet.
-  </p>
-</aside>
+custom audio processor with JavaScript code. Please take a look at the [live
+demos](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/) on
+GitHub or [the instruction](#experimental) on how to use this feature.
+
+### Background: ScriptProcessorNode
+
+Audio processing in Web Audio API runs in a separate thread from the main
+UI thread, so it runs smoothly. To enable custom audio processing in
+JavaScript, the Web Audio API proposed a ScriptProcessorNode which used
+event handlers to invoke user script in the main UI thread.
+
+There are two problems in this design: the event handling is asynchronous
+by design, and the code execution happens on the main thread. The former
+induces the latency, and the latter pressures the main thread that is
+commonly crowded with various UI and DOM-related tasks causing either UI
+to "jank" or audio to "glitch". Because of this fundamental design flaw,
+<code>ScriptProcessorNode</code> is deprecated from the specification and
```

Comment:
You can wrap these in backticks (`) instead of using code

Reply:
Oh, I did not change the part. The original text had this.